performance.rst

```diff
@@ -119,7 +119,9 @@ The preload file path is the same as the compiled service container but with the
 .. code-block:: ini
 
     ; php.ini
-    opcache.preload=/path/to/project/var/cache/prod/srcApp_KernelProdContainer.preload.php
+    opcache.preload=/path/to/project/.preload.php
```

Comment:
Should we add a versionadded for 4.4.14 explicitly? 🧐

Reply:
Not sure: this note would become useless quite fast as ppl upgrade.